wp-table-migrations.php

```diff
@@ -180,4 +179,44 @@ public static function getDirName(){
 
     TableMigrations::getInstance();
 
-});
\ No newline at end of file
+});
+
+/**
+ * Load Factory Migrations.
+ *
+ */
+function load_factory_migrations() {
+	if ( ! defined( 'WP_CLI' ) || defined( 'WP_CLI' ) && ! WP_CLI ) {
+		return;
+	}
+
+	$elements = glob( ABSPATH . '/migrations/*.php' , 0 );
+
+	foreach ( $elements as $migration_path ) {
+		require $migration_path;
+	}
+}
+
+add_action( 'table_migrations_loaded', 'load_factory_migrations' );
+
+/**
+ * Wrapper for the switch_to_blog function that applies a blog switch based
+ * on a unique meta field saved for each site of the factory.
+ *
+ * @param string $blog_name Tag value used to search for the site.
+ *
+ * @return bool
+ */
+function collabra_switch_to_blog( $blog_name ) {
+	foreach ( get_sites() as $site ) {
```

Comment:
Use `get_sites` have support to meta queries.
So use it here instead of looping all sites.

```php
'meta_key'     => 'site_tag',
'meta_value'   => 'tag_name',
```